### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by Sarahhud**

### CODE REVIEW

File: capstone-app/src/app/modules/history/wiki-results.service.ts [hunk 1]
```diff
@@ -53,17 +53,23 @@ export class WikiResultsService {
       const startIndex = firstPartOfString.indexOf('</h2>', 0);
       let middleOfString = firstPartOfString.substring(startIndex, endIndex);
       const paragraphs = middleOfString.split('<p>');
-      if (paragraphs.length > 6){
-        middleOfString = paragraphs.slice(0,6).join('');
+      console.log(paragraphs);
+      if (paragraphs.length > 9) {
+        if(paragraphs[0].startsWith("</h2")) {
+          middleOfString = paragraphs.slice(1,9).join('');
+        }
```
Comment (NicholasCobbFromGoogle) [inline capstone-app/src/app/modules/history/wiki-results.service.ts]: If you want to display the first nine paragraphs and you need to omit the "</h2" paragraph, should we make this slice (1, 10)?

**Review by NicholasCobbFromGoogle — APPROVED** [review]

File: capstone-app/src/app/modules/history/wiki-results.service.ts [hunk 2]
```diff
@@ -53,17 +53,23 @@ export class WikiResultsService {
       const startIndex = firstPartOfString.indexOf('</h2>', 0);
       let middleOfString = firstPartOfString.substring(startIndex, endIndex);
       const paragraphs = middleOfString.split('<p>');
-      if (paragraphs.length > 6){
-        middleOfString = paragraphs.slice(0,6).join('');
+      console.log(paragraphs);
```
Comment (billyporter) [inline capstone-app/src/app/modules/history/wiki-results.service.ts]: remove console log

**Review by billyporter — APPROVED** [review]

File: capstone-app/src/app/modules/history/wiki-results.service.ts [hunk 3]
```diff
@@ -53,17 +53,23 @@ export class WikiResultsService {
...
+      if (paragraphs.length > 9) {
+        if(paragraphs[0].startsWith("</h2")) {
```
Comment (mbehrlich) [inline capstone-app/src/app/modules/history/wiki-results.service.ts]: Nit: add space after "if"

Comment (mbehrlich) [reply capstone-app/src/app/modules/history/wiki-results.service.ts]: And use single quotes ' ' instead of double quotes " "

**Review by mbehrlich — APPROVED** [review]

File: capstone-app/src/app/modules/history/wiki-results.service.ts (hunk 1 above)
Comment (Sarahhud) [reply capstone-app/src/app/modules/history/wiki-results.service.ts]: Ah, yes. Good point!

**Review by smontana30 — APPROVED** [review]

**Pull request merged**